docs/content/docs/patterns/observability/tracing.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Tracing"
+description: "Tracing"
+type: docs
+---
+
+# Tracing
+Open Service Mesh (OSM) allows optional deployment of Jaeger for tracing. 
+
+## Jaeger
+[Jaeger](https://www.jaegertracing.io/) is an open source tracing system used for monitoring and troubleshooting distributed systems. It can be deployed with OSM as a new instance or you may bring your own instance. 
+
+### Automatic bring up
+By default, Jaeger deployment and tracing as a whole is disabled.
+
+Jaeger can be automatically deployed through either by using the `--deploy-jaeger` OSM CLI flag at install time or by toggling the `deployJaeger` value in `values.yaml`.
+
+Similarly, tracing can be enabled and customized during installation (`tracing` section in `values.yaml`) or at runtime by editing `osm-configmap`. Tracing can be enabled, disabled and configured any time to support BYO scenarios.
```

Comment:
@eduser25 I've made the edits you suggested and reworded this section a little for clarity. I'm a little confused about whether we're treating Jaeger and tracing as two separate things? It sounds like that in this line

Reply:
they are separate things.
It is like so as it helps deploy a tracing application that are able to receive in zipkin protocol, not just Jaeger - can just be anything that can understand zipkin proto.

Reply:
Hm so below where it references specifically Jaeger in the BYO section, is that wrong? Should I change it to say an already running instance of....something else?
```
The following section documents the additional steps needed to allow an already running instance of Jaeger to integrate with your OSM control plane.

Prerequisites
* A running Jaeger instance
```